library/std/src/personality/dwarf/eh.rs

```diff
@@ -184,7 +185,7 @@ unsafe fn read_encoded_pointer(
     };
 
     if encoding & DW_EH_PE_indirect != 0 {
-        result = *(result as *const usize);
+        result = *ptr::from_exposed_addr::<usize>(result);
```

Comment:
I hope this can only happen if we previously had the `DW_EH_PE_pcrel` case above and exposed the pointer...

Reply:
Who knows, but given that `as *const usize` is essentially semantically equivalent to `from_exposed_addr::<usize>(...)`, this shouldn't add additional UB or anything if that's not how it is.

Reply:
This could add UB only if the code relied on the `round_up` stuff above also exposing the pointer. But that did look like a slam-dunk case for `with_addr` so I went with that. (Actually it would be an awesome case for `map_addr` except that it would need `?` inside the closure so of course that doesn't work...)

Reply:
> This could add UB only if the code relied on the round_up stuff above also exposing the pointer

That would be incredibly goofy and surprising, so I'm going to assume it's not the case.